dotCMS/src/main/java/com/dotcms/util/FunctionUtils.java

```diff
@@ -74,6 +74,28 @@ public static boolean ifOrElse(final Supplier<Boolean> booleanSupplier, final Ca
         return ifOrElse(booleanSupplier.get(), onTrueCallback, onFalseCallback);
     } // ifOrElse.
 
+    /**
+     * Tries to get the value, if some error happens it is not logged or anything, just quietly returned the defaultValue.
+     * @param delegate {@link ReturnableDelegate} delegate to call
+     * @param defaultValue T default value in case of error
+     * @param <T>
+     * @return T
+     */
+    public static <T> T getValueOrDefault (final ReturnableDelegate<T> delegate, final T defaultValue) {
+
+        T returnValue = defaultValue;
+
+        try {
+
+            returnValue = delegate.execute();
+        } catch (Throwable e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)

Reply:
be quietly is the idea of the method, actually